modules/gapi/src/api/gmat.cpp

```diff
@@ -137,6 +137,13 @@ template<typename M> inline bool canDescribeHelper(const GMatDesc& desc, const M
 }
 } // anonymous namespace
 
+bool GMatDesc::isVectorPoints(int n, int ddepth) const
```

Comment:
Ok. This function checks if GMatDesc contains a vector or not.
Possible cases (CHW):
- n x L x 1;
- n x 1 x L;
- 1 x L x n
where n - point (non cv::Point @OrestChura ??) dimension, L - number of points.

@AsyaPronina  I think, it looks like truth.

Reply:
Yes, all of it is correct. Checks if Mat (GMatDesc) contains n-dimensional points.

Reply:
I don't understand why -1 can be passed as ddepth here and why this is ddepth? Is it really destination depth?

Also, is this method really needed to be in GMatDesc?
May be we can introduce just single function in the tests which will test GMatDesc to be vector of points?
Is it really truth and vector of points will be required to be either 1xH or Wx1? May be some kernels will require stronger limitations and expect vector to be strictly 1xH?

Also why isVectorPoints and not isPointsVector?

Reply:
`ddepth` can be passed to check for a certain depth of Mat; if -1 is passed, depth check is not occured.

I finally agree, his shouldn't be a method as there's no real need of it. Changed to a separate function in anonymous namespace (next to meta check for findContours)

The check follows the `Mat::checkVector` method, all the requirements on sizes are from there

The name is changed, thanks

/cc @TolyaTalamanov please review